### PR TITLE
Filter rules by age first part of #284

### PR DESCRIPTION
**Pull request by iphands**
First part of frontend change for #284

### CODE REVIEW

**Comment by iphands** [comment]
This PR is based on #286 at the moment.

**Comment by Kinlaw** [comment]
@iphands Is the backend setup to filter by published_age?

**Comment by Kinlaw** [comment]
My bad. I see the PR for it in the api now.

**Comment by iphands** [comment]
@Kinlaw dude, I lie lie

I think the:
```javascript
let option = $location.search()[Events.filters.impact] ?  $location.search()[Events.filters.impact]  : FilterService.getImpact();
```

This was important. In the case where a page change happens from in the app and not a reload. i.e. using ui-sref="app.rules({age: 15}) from say the Overview page...

In that case the filter service is already instantiated and _age is set to 0 (because the previous page does not have this param).  But when I refresh the /rules?age=15 page I get what I want because the filter service is initialised and looks at location.

Im gonna change this back to what we had, but I think longer term it would be neat to have things be reparsed automatically w/o the need for this in each controller.

**Comment by Kinlaw** [comment]
Depends on https://github.com/RedHatInsights/insights-api/pull/59

File: app/js/services/filter.service.js [hunk 1]
```diff
@@ -619,6 +629,11 @@ function FilterService(
             });
         }
 
+        //age
+        if (includeParam('age') && _age !== 0) {
```
Comment (Kinlaw) [inline app/js/services/filter.service.js]: this should be 
`if (includeParam('age') && _age !== '0') {`

the age is being stored as a string

Comment (iphands) [reply app/js/services/filter.service.js]: Nice catch! Thanks Josh.

**Comment by iphands** [comment]
Removed waiting on API

**Pull request merged**